sbin/common/common.sh

```diff
@@ -173,7 +173,25 @@ function setBootJdk() {
     fi
 
     echo "Guessing JDK_BOOT_DIR: ${BUILD_CONFIG[JDK_BOOT_DIR]}"
-    echo "If this is incorrect explicitly configure JDK_BOOT_DIR"
+    echo "If this is incorrect explicitly configure JDK_BOOT_DIR using --jdk-boot-dir"
+
+    # Calculate version number of boot jdk
+    export BOOT_JDK_VERSION_NUMBER=$(${BUILD_CONFIG[JDK_BOOT_DIR]}/bin/java -XshowSettings:properties -version 2>&1 | awk '/java.specification.version/{gsub(/1\./,"",$3);print $3}')
```

Comment:
Quoting necessary of `${BUILD_CONFIG[JDK_BOOT_DIR]}/bin/java`

Reply:
I get an error similar to the below without the quotes
```
/Users/morgan/Documents/Repos/openjdk-build/sbin/common/common.sh: line 183: BUILD_CONFIG[JDK_BOOT_DIR]/bin/java: No such file or directory
```